meta-ostro/conf/distro/ostro.conf

```diff
@@ -22,6 +22,20 @@ SDK_UPDATE_URL = "${OSTRO_SDK_UPDATE_URL}"
 # The default setting can be overridded in local.conf
 OSTRO_SDK_UPDATE_URL ??= "https://download.ostroproject.org/builds/ostro-os/latest/sdk-data/${MACHINE}"
 
+# The information about where to find version information and actual
+# content is needed in several places:
+# - the swupd client in the image gets configured such that it uses that as default
+# - swupd server needs information about the previous build
+#
+# The version URL determines what the client picks as the version that it updates to.
+# The content URL must have all builds ever produced and is expected to also
+# have the corresponding version information.
+SWUPD_VERSION_URL ?= "https://download.ostroproject.org/updates/ostro-os/milestone/${MACHINE}/${SWUPD_IMAGE_PN}"
+SWUPD_CONTENT_URL ?= "https://download.ostroproject.org/updates/ostro-os/builds/${MACHINE}/${SWUPD_IMAGE_PN}"
```

Comment:
builds/ directory is something that can be removed at any point. only hierarchy over milestone/ is known to be preserved always, so content url also should be pointing to the place where version is taken from.